rest-service/manager_rest/rest/resources_v3_1/status.py

```diff
@@ -136,34 +141,44 @@ def get(self):
         return {'status': status, 'services': services}
 
     def _check_supervisord_services(self, services):
+        statuses = []
+        supervisord_services = get_system_manager_services(
+            SUPERVISORD_BASE_SERVICES,
+            SUPERVISORD_OPTIONAL_SERVICES
+        )
+        for name, display_name in supervisord_services.items():
```

Comment:
If we can collapse the service listings together then we can presumably refactor this with the _check_systemd_services since the logic is basically the same?

Reply:
Based on my comment added above, the change requries to change service file name on the manager side as well as the services that we are not override/configure like nginx/postgresql

Reply:
Did not combine the two logic into one method as it will increase complexity somehow as both supervisord and systemd used different interfaces for getting data about the services and some pre call is needed for systemd and not for supervisord.